### DISCUSSION TITLE
DB connections list. Hotkey for quick Disconnect

### DISCUSSION THREAD

**Discussion by ghYura** [dbeaver/dbeaver#19086] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when among multiple connections have to open context menu and press menu item 'Disconnect'

**Describe the solution you'd like**
Add Hotkey for quickly disconnect  '**Ctrl + Backspace**' (since simple key 'Backspace' assigned for navigating to upper menu-list item)
(see screenshot)

**Describe alternatives you've considered**
none

Thank you.

**Additional context**
![image](https://user-images.githubusercontent.com/42974628/219357994-fab7d274-fd34-4bd7-b6ea-eb200f915164.png)

**Comment by ghYura** [comment]
Trivial feature

**Comment by ghYura** [comment]
Probably, 'Ctrl+Shift+Backspace' hotkeys - for total all disconnection

**Comment by ghYura** [comment]
Not sure, if proposed hotkey commands contradict with some other commands inside other focused active window(s) and/or tab(s) in this DB manager...

**Comment by E1izabeth** [comment, marked as answer]
You can assign any shortcut you want to this command in preferences
![image](https://user-images.githubusercontent.com/28875055/219380681-1188e591-a85c-48b5-b30a-594adf6c5674.png)

**Comment by ghYura** [comment]
@E1izabeth 
Thank you. Indeed feature present.
Closing issue.
